packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -212,6 +218,9 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
 
   Widget _buildChildren(BuildContext context, Widget? child) {
     final Color borderSideColor = _borderColor.value ?? Colors.transparent;
+    final Widget trailing = widget.trailing ?? const Icon(Icons.expand_more);
+    final bool rotatingTrailing =
+        widget.rotatingTrailing ?? widget.trailing == null;
```

Comment:
Why did this end up on a new line? Did you reformat the code?

Reply:
only that line (which is newly introduced btw). I think it's passing [the 80 chars limit](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters), but worth to double-check.

Reply:
lol, it's 81 chars

Reply:
I could change `rotatingTrailing` to `rotateTrailing` everywhere. wdyt?

Reply:
Just drop the variable type, at least according to the Dart recommendation it shouldn't be there:

```dart
final trailing = widget.trailing ?? const Icon(Icons.expand_more);
final rotatingTrailing = widget.rotatingTrailing ?? widget.trailing == null;
```

Reply:
weird, I've added it because [lint complained about it](https://dart-lang.github.io/linter/lints/always_specify_types.html). any idea why?

Reply:
Ahh, so the Flutter team has taken that road... well, you can set this in the `analysis_options.yaml` file, and at least according to the Google recommended rules, types should be ommitted. I guess the Flutter team turned on that rule.

Ohh well, back to square 1 :) Your suggestion sounds good.